src/components/paper-time-input.js

```diff
@@ -167,6 +171,28 @@ export class PaperTimeInput extends PolymerElement {
           always-float-label$="[[alwaysFloatInputLabels]]"
           disabled="[[disabled]]"
           hidden$="[[!enableSecond]]"
+        >
+          <span hidden$="[[!enableMillisecond]]" suffix slot="suffix">:</span>
+        </paper-input>
+
+        <!-- Millisec Input -->
+        <paper-input
```

Comment:
We should really make this element a LitElement, it is so complicated now...

Reply:
@bramkragten That is not something I can manage before the beta and I think this fix should be in. I tackled all your other remarks.

Reply:
Should not be in this PR anyway